### PR TITLE
feat: Use docker to build profile-launcher

### PR DESCRIPTION
**Pull request by antoniomtz**
## PR Checklist

- [ x] Added label to the Pull Request for easier discoverability and search
- [x ] Commit Message meets guidelines as indicated in the URL https://github.com/intel-retail/automated-self-checkout/blob/main/CONTRIBUTING.md
- [ ] Every commit is a single defect fix and does not mix feature addition or changes
- [ ] Unit Tests have been added for new changes
- [ ] Updated Documentation as relevant to the changes
- [ ] All commented code has been removed
- [ ] If you've added a dependency, you've ensured license is compatible with repository license and clearly outlined the added dependency.
- [ ] PR change contains code related to security
- [ ] PR introduces changes that breaks compatibility with other modules (If YES, please provide details below)

## What are you changing?
Building process of profile-launcher

## Issue this PR will close

close: #293

## Test Instructions if applicable

1. make build-profile-launcher

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -36,7 +36,7 @@ clean-simulator:
 	./clean-containers.sh camera-simulator
 
 build-profile-launcher:
-	@cd ./configs/opencv-ovms/cmd_client && $(MAKE) build
+	@cd ./configs/opencv-ovms/cmd_client && docker build --target bin --output=${PWD}/configs/opencv-ovms/cmd_client .
```
Comment (jim-wang-intel) [inline Makefile:39]: we can update makebuild target in the other side and just call make build here the same way. i.e: refactor the make build target in another makefile so they are all encapsulated in that place

Comment (jim-wang-intel) [reply Makefile:39]: this one i meant: https://github.com/intel-retail/automated-self-checkout/blob/main/configs/opencv-ovms/cmd_client/Makefile

**Review by brian-intel — APPROVED** [review]

File: configs/opencv-ovms/cmd_client/Makefile [hunk 2]
```diff
@@ -4,9 +4,13 @@
 .PHONY: build unit-test
```
Comment (jim-wang-intel) [inline configs/opencv-ovms/cmd_client/Makefile:4]: fix the .PHONY to have build-binary

**Review by brian-intel — APPROVED** [review]
LGTM

**Review by jim-wang-intel — APPROVED** [review]
LGTM

**Pull request merged**